src/backend/src/utils/auth.utils.ts

```diff
@@ -78,3 +78,5 @@ export const getCurrentUser = async (res: Response): Promise<UserWithTeam> => {
   if (!user) throw new NotFoundException('User', userId);
   return user;
 };
+
+export type UserWithTeam = User & { teams: Team[] };
```

Comment:
hmm

Reply:
this is from a different PR i just moved it to this file